src/KnowledgeQuiz.tsx

```diff
@@ -42,35 +53,104 @@ const fieldsetStyle = css`
 export const KnowledgeQuizAtom = ({
     id,
     questions,
-}: QuizAtomType): JSX.Element => (
-    <form data-atom-id={id}>
-        {questions.map((question, idx) => (
-            <Question
-                key={question.id}
-                id={question.id}
-                number={idx + 1}
-                text={question.text}
-                imageUrl={question.imageUrl}
-                answers={question.answers}
-            />
-        ))}
-    </form>
-);
+    resultGroups,
+    sharingIcons,
+}: QuizAtomType): JSX.Element => {
+    const [selectedGlobalAnswers, setSelectedGlobalAnswers] = useState<{
```

Comment:
yeah this was kinda difficult to decide, we technically have 2 points where we define selected answers. Once in `KnowledgeQuizAtom` and in `Question` components. The global object of selected answers contains all the selection made for the quiz, we use that to determin if all questions have been answered.

I guess we could maybe call this `quizSelection` because it is the selection made within the quiz?